fonts.user.styl

```diff
@@ -7,7 +7,7 @@
 @supportURL   https://github.com/vednoc/custom-fonts/issues
 @updateURL    https://userstyles.world/api/style/85.user.css
 @preprocessor stylus
-@version      0.5.0
+@version      0.6.0
```

Comment:
Remove version bump because I'll do it in a separate commit.

Reply:
ok